### PR TITLE
w3sper: add lux module for parsing and formatting in Dusk

### LINKED ISSUE #3050: w3sper: easily convert between DUSK and LUX units.
In the checklist of functionalities that the SDK should provide for Exchanges, there is a clear need for conversion between DUSK and LUX units.

While the SDK operates exclusively in LUX, for user interactions, the SDK should support:

 - Formatting LUX as DUSK: Displaying LUX values in a human-readable DUSK format.
 - Parsing DUSK to LUX: Converting user-inputted DUSK values into the LUX representation for internal use.

Purpose:

DUSK serves purely as a display format for LUX values, analogous to how currencies are formatted for readability (e.g., adding decimal places). This functionality ensures that users, who are more likely to work with DUSK as an intuitive unit, can seamlessly interact with the SDK while maintaining the precision and efficiency of LUX for backend operations.

### PR DESCRIPTION
**Pull request by ZER0**
Resolves #3050

### CODE REVIEW

File: w3sper.js/tests/lux_test.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  );
+  assert.equal(lux.parseFromDusk("9007199.254740993"), 9_007_199_254_740_993n);
+  assert.equal(lux.parseFromDusk("10.000000001"), 10_000_000_001n);
+  assert.equal(lux.parseFromDusk("3141592.653589793"), 3_141_592_653_589_793n);
+
+  assert.equal(
+    lux.parseFromDusk("123456789012345678901.23456789"),
+    123_456_789_012_345_678_901_234_567_890n,
+  );
+});
```
Comment (Daksh14) [inline w3sper.js/tests/lux_test.js:47]: I think we should include tests for negative integers, and out of bounds check? Overflow while arithmetic?

**Review by Daksh14 — COMMENTED** [review]
Only one question about the test

**Review by ascartabelli — APPROVED** [review]

**Review by ascartabelli — APPROVED** [review]

File: w3sper.js/tests/lux_test.js (hunk 1 above)
Comment (ZER0) [reply w3sper.js/tests/lux_test.js:77]: I added in the documentation the reason why we won't deal with non-negative values, and also add the exceptions / tests for such use case.

About overflow, is not a problem per se; this is JS not Rust, it handles such things differently.

**Review by ascartabelli — APPROVED** [review]
